### ISSUE TITLE
Create a embracing and welcoming StandardView if no routes are present

### ISSUE THREAD

**Issue by sorenmalling** [neos/flow-development-collection#2110]
### Description

The current "Standard View" looks like this

![image](https://user-images.githubusercontent.com/531918/92450708-0b108880-f1bc-11ea-91d3-018f4a3c7752.png)

and is resolved via this controller 

https://github.com/neos/flow/blob/master/Classes/Mvc/Controller/StandardController.php

We can create a more embracing look and feel of this and provide help to get going (similar to the work on the new Neos.Welcome package)

**Comment by mrimann** [comment]
I struggle to “reproduce” the problem with a fresh flow-developmend-distribution, when I call it in a browser, it first showed the nice fresh Welcome-Package content (fine). Once I removed the Welcome package, I only get a 404 **“Sorry, the page you requested was not found.”** view but not this view shown above as a screenshot (the same old visual style, but not the same content).

As far as I could track it down, this StandardController is not getting found because there’s no Route matching. I now got it to show this welcome page out Flow by setting the following config to include Flow’s Routes:

```
Neos:
  Flow:
    mvc:
      routes:
        'Neos.Flow':
          position: 'start'
```

So I wonder whether this is really worth the effort to fix this issue, when this view is basically not rendered to the user at all (without fiddling with the config first). Or am I overlooking something?

Would be cool to hear your thoughts about this and maybe a pointer on how to proceed

**Comment by albe** [comment]
Right, you're not wrong that this only is visible when someone enables the Flow route and doesn't have an own view already. The StandardView is part of Flow and it would be nice to have that more aesthetically matching with the new Welcome package. Surely not anything urgent, but a little detail we could improve.
